ee/ucp/authorization/isolate-nodes.md

```diff
@@ -226,20 +226,42 @@ Select the **ops-nodes** namespace, and create a `Full Control` grant for the
 
 ### Link the namespace to a node collection
 
-The last step is to link the Kubernetes namespace the `/Prod` collection.
-
-1.  Navigate to the **Namespaces** page, and find the **ops-nodes** namespace
-    in the list.
-2.  Click the **More options** icon and select **Link nodes in collection**.
+The last step is to link the Kubernetes namespace to the `/Prod` collection. Namespaces are associated with a node collection using the namespace definition information provided in a configuration and an annotation file:
+    
+#### Configuration file
+The configuration file specifies backend behavior in json or yaml using the following format:
```

Comment:
So from my understanding based on https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#configuration-file-format-1, there are two methods to accomplish isolating nodes to a namespace. Format # 1 outlines how to do it all at once from a single configuration file or Format #1 specify it within every Namespace definition. TBH I'm recommending the second format as it's more straight forward and can be done when the namespace is created. But you can link this kubernetes document in our doc and say here are the two formats.